fee_allocator/fees_collected/v3_fees_2025-11-20_2025-12-04.json

```
@@ -0,0 +1,9 @@
+{
+  "optimism": 215508895,
+  "mainnet": 17729846623,
```

this is incorrect, you also subtracted optimism from the final amount
```suggestion
  "mainnet": 17945355518,
```